nautobot/extras/templates/extras/inc/job_tiles.html

```diff
@@ -0,0 +1,72 @@
+{% load helpers %}
+
+{% block extra_styles %}
+    <style>
+        .panel.panel-default {
+            border: none;
+            box-shadow: none;
+        }
+    </style>
+{% endblock %}
+
+<div class="tiles">
+    {% for row in table.page.object_list|default:table.rows %}
+        <article class="tile{% if not perms.extras.run_job or not row.record.runnable %} disabled{% endif %}">
+            <header class="tile-header">
+                <h3>{{ row.record.grouping|bettertitle }}</h3>
+
+                <a href="{% url 'extras:job_run' pk=row.record.pk %}" class="btn btn-primary btn-xs" title="Run/Schedule" {% if not perms.extras.run_job or not row.record.runnable %}disabled="disabled"{% endif %}>
+                    <i class="mdi mdi-play" aria-hidden="true"></i>
+                </a>
+
+                {% comment "TODO(norbert-mieczkowski-codilime): use the same buttons (actions) in tables and tiles when tables support dropdown menus" %}
+                    {% for item in row.items %}
+                        {% if item.0.name == "actions" %}
+                            {{ item.1 }}
+                        {% endif %}
+                    {% endfor %}
+                {% endcomment %}
+                <div class="btn-group">
+                    <button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Show more">
```

Comment:
Maybe `Additional Links` or `Actions` would be better than `Show more`? Not sure.

Reply:
@lampwins, which text would you prefer?

Reply:
I like `Actions` and I think we even call these `action_buttons` in other areas of the app.